console/v1/types_console_plugin.go

```diff
@@ -48,6 +48,26 @@ type ConsolePluginSpec struct {
 	// i18n is the configuration of plugin's localization resources.
 	// +optional
 	I18n ConsolePluginI18n `json:"i18n"`
+	// csp is a list of Content Security Policy directives for the plugin.
```

Comment:
For the final doc, we'll want more detail about what this is and why you'd need to change it. Examples would be helpful.